ml_perf/flags/9/train.flags

```diff
@@ -13,6 +13,6 @@
 --lr_rates=0.16
 --lr_rates=0.016
 --lr_rates=0.0016
---lr_boundaries=25000
---lr_boundaries=37500
+--lr_boundaries=10000
```

Comment:
did they get to the 10k boundary?

Reply:
Just barely. These boundaries don't really make sense but I want to push the flags files without any changes to make sure I don't accidentally break something.